conf/broker.conf

```diff
@@ -327,6 +327,14 @@ brokerPublisherThrottlingMaxMessageRate=0
 # (Disable byte rate limit with value 0)
 brokerPublisherThrottlingMaxByteRate=0
 
+# Max Rate(in 1 seconds) of Message allowed to dispatch from a broker if broker dispatch rate limiting enabled
+# (Disable message rate limit with value 0)
+brokerDispatchThrottlingMaxMessageRate=0
+
+# Max Rate(in 1 seconds) of Byte allowed to dispatch from a broker if broker dispatch rate limiting enabled.
+# (Disable byte rate limit with value 0)
+brokerDispatchThrottlingMaxByteRate=0
```

Comment:
```suggestion
dispatchThrottlingRateInByte=0
```

Reply:
And we already have some `dispatchThrottling` related configurations such as `dispatchThrottlingRatePerTopicInByte`, it's better to put them together and follow the same name pattern

Reply:
Done